docs/installation.md

```diff
@@ -0,0 +1,77 @@
+## Prerequisites
+
+Ensure you have Python version 3.9 or above installed on your system.
+
+We recommend creating and activating a Python virtual environment using [`venv`](https://docs.python.org/3/library/venv.html) before installing `nemos`. This practice helps manage dependencies and avoid conflicts with other Python packages.
+
+For `venv`, create your virtual environment in a specific directory. This example uses `~/python_venvs/nemos` for Linux/Mac and `C:\Users\username\python_venvs\nemos` for Windows; in general, free to use whatever directory works best for you.
```

Comment:
People will think that `Users\username` is the path they should use, I'd replace it with `%HOMEPATH%`, which I think is the windows equivalent of `$HOME` ([stackoverflow](https://superuser.com/questions/1319038/how-to-set-home-variable-so-that-cmd-find-the-path))

@EricThomson  can you check that the commands in this section work with `%HOMEPATH%\python_venvs\nemos\`, just like that?

Reply:
I checked and it works